spec/helper/MockHelper.ts

```diff
@@ -241,7 +241,7 @@ interface EventEmitter {
 
   fireAdErrorEvent(): void;
 
-  fireVideoPlaybackQualityChangedEvent(bitrate: number): void;
+  fireVideoPlaybackQualityChangedEvent(bitrate: number, frameRate: number): void;
```

Comment:
Please make `frameRate` optional, so we can test events with and without `frameRate` being available.